worker/src/utils/migrate.ts

```diff
@@ -24,7 +24,7 @@ export const migrateAnalyticsData = (data: any): AnalyticsData => {
       reports_statistics: 0,
       extended_data_from: 0,
       versions: {},
-      operating_system: { boards: {}, versions: {} },
+      operating_system: { boards: {}, versions: {}, board_versions: {} },
```

Comment:
Can you extend this file with the new dictionary as well?
https://github.com/home-assistant/analytics.home-assistant.io/blob/dev/site/src/prometheus/base.liquid#L49